docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+## Datasources
+
+![Datasource overview](./images_overview/datasource_flowchart.png)
+
+Datasources connect GX to your source data.  Your source data could be CSV files in a folder, a PostgreSQL database hosted on AWS, or any combination of numerous data formats and environments.  Regardless of your source data's format and where it resides, Datasources provide GX with a unified API for working with it.
+
+- [Datasource tutorials](/docs/guides/connecting_to_your_data/connect_to_data_lp)
+
+### Data Assets and Batches
+
+Data Assets are collections of records within a Datasource.  While a Datasource tells GX how to connect to your source data, Data Assets tell GX how to organize that data.  For instance, you could define a Data Asset for a SQL Datasource as all the records within a specific table.  For a File Datasource, you could define a Data Asset as all of the `.csv` files in a specific subfolder.
+
+Data Assets allow you to organize your data into collections of records that do not necessarily conform to the schema of your source data.  For instance, in a SQL Datasource you can use a Query Data Asset to combine an arbitrary set of records into a single Data Asset.  Likewise, in a File Datasource you can use regular expressions (regex) to group a specific set of files into a single Data Asset based on their filenames.
+
+Data Assets can be further partitioned into Batches.  Batches are unique subsets of records within a Data Asset.  For example, say you have a Data Asset in a SQL Datasource that consisted of all records from last year in a given table.  You could then partition those records into Batches of data that correspond to the records for individual months of that year.
+
+Using Batches to further partition a Data Asset is optional.  However, having a list for Batches in your Data Asset is not.  This is because GX uses Batches to specify the data to work with.  By default, if you do not further partition the data in a Data Asset it will contain a list with only one Batch.  In that case, the single Batch will consist of all the records in that Data Asset.
+
+- [Data Asset and Batch tutorials](/docs/guides/connecting_to_your_data/manage_data_assets_lp)
+
+### Batch Requests
+
+A Batch Request specifies one or more Batches within the Data Asset.  Batch Requests are the primary way of retrieving data for use in GX.  Because Batch Requests can specify multiple Batches they provide significant flexibility in how you utilize the data in a single Data Asset.
+
+As an example, GX can automate the process of running statistical analyzes for multiple Batches of data.  This is possible because you can provide a Batch Request that corresponds to multiple Batches in a Data Asset.  Alternatively, you can specify a single Batch from that same Data Asset so that you do not need to re-run the analyzes on all of your data when you are only interested in a single subset.
```

Comment:
I'm a bit confused by the next two paragraphs. I think you're trying to say because we can retrieve multiple batches within a single batch request, we can run more sophisticated validations, such as expectations on some data across a period of time?